### PR TITLE
Add schema deployment

### PR DESCRIPTION
**Pull request by southeo**
Add yaml files for schemas.dissco.tech deployment

### CODE REVIEW

File: schemas/schemas-route.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - match: Host(`schemas.dissco.tech`)
+      kind: Rule
+      services:
+        - name: dissco-schemas
+          namespace: default
+          port: 8080
+      middlewares:
+        - name: dissco-schemas-cors-headers
+---
+apiVersion: traefik.io/v1alpha1
```
Comment (samleeflang) [inline schemas/schemas-route.yaml:30]: Is this really needed? You might be right to put it here. What is there looks good.

File: schemas/schemas-route.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: dissco-schemas-manager-route
+  namespace: default
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - match: Host(`schemas.dissco.tech`)
```
Comment (samleeflang) [inline schemas/schemas-route.yaml:21]: I will add a record in the DNS for this

**Review by samleeflang — APPROVED** [review]

**Review by samleeflang — COMMENTED** [review]
Oh I just remembered maybe we should prefix this with `dev-` as well as this is not the production version...

File: schemas/schemas-route.yaml (hunk 1 above)
Comment (southeo) [reply schemas/schemas-route.yaml:30]: Not 100% sure if this is necessary, this is from the other routing files. Better safe than sorry?

**Pull request merged**